src/Compilers/CSharp/Test/Symbol/Symbols/Source/RecordTests.cs

```diff
@@ -225,6 +225,9 @@ record C(int X, int Y)
 }
 ");
             comp.VerifyDiagnostics(
+                // (4,17): error CS8872: 'C.Equals(C)' disallows overriding and containing 'record' is not sealed.
```

Comment:
nit: the wording might be more helpful as "`C.Equals(C)` must allow overriding because the containing record is not sealed"

Reply:
> nit: the wording might be more helpful as "C.Equals(C) must allow overriding because the containing record is not sealed"

Rikki already suggested that and I already incorporated this change in the next PR

---
In reply to: [451903816](https://github.com/dotnet/roslyn/pull/45647#discussion_r451903816) [](ancestors = 451903816)